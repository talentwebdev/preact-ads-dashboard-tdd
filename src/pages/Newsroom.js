import React from 'react'

function Newsroom() {
    return (
        <div>
            This is a newsroom page
        </div>
    )
}

export default Newsroom
