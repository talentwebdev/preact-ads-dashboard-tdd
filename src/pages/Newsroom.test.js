import {shallow} from 'enzyme';
import Newsroom from './Newsroom';

describe('Newsroom', () => {
    let newsroomContainer;

    beforeEach(() => newsroomContainer = shallow(<Newsroom />));

    it('should render the div element', () => {
        expect(newsroomContainer.find('div')).toBeTruthy();
    });
});