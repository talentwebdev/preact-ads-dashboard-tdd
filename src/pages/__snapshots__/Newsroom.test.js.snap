// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Newsroom renders Newsroom correctly 1`] = `ShallowWrapper {}`;
